src/com/janek/basics/CalcGold.java

```diff
@@ -0,0 +1,101 @@
+package com.janek.basics;
+
+import static com.janek.basics.math.MathUtil.*;
+import static java.lang.Double.parseDouble;
+import static java.lang.String.format;
+import static java.lang.System.exit;
+import static java.lang.System.out;
+
+public class CalcGold {
+
+    private static final String INPUT_PLUS_SYMBOL = "+";
+    private static final String INPUT_PLUS_STRING = "dodać";
+    private static final String INPUT_MINUS_SYMBOL = "-";
+    private static final String INPUT_MINUS_STRING = "odjąć";
+    private static final String INPUT_X_SYMBOL = "x";
+    private static final String INPUT_X_STRING = "razy";
+    private static final String INPUT_SLASH_SYMBOL = "/";
+    private static final String INPUT_SLASH_STRING = "podzielić-przez";
+
+    public static final int VALIDATION_EXIT_CODE = 1;
+
+    public static void main(String[] args) {
+        validateArgsNumber(args);
+        validateSymbol(args);
+        validateNumber(args);
+
+        double result = calculateResult(args);
```

Comment:
You can also use `var` java 11 keyword to declare the variable:

```
double result = calculateResult(args);
```
is the same as 
```
var result = calculateResult(args);
```